projects/plugins/jetpack/modules/videopress/class.videopress-xmlrpc.php

```diff
@@ -7,204 +9,15 @@
-				$media_id,
-				array(
-					'original' => array(
-						'url' => $media_item['url'],
-					),
-				)
-			);
-
-			$media_item['post'] = get_post( $media_id );
-		}
-
-		return array( 'media' => $media );
-	}
-
-	/**
-	 * Update VideoPress metadata for a media item.
-	 *
-	 * @param array $request Media item to update.
-	 *
-	 * @return bool
-	 */
-	public function update_videopress_media_item( $request ) {
-		$this->authenticate_user();
-
-		$id     = $request['post_id'];
-		$status = $request['status'];
-		$format = $request['format'];
-		$info   = $request['info'];
-
-		$attachment = get_post( $id );
-		if ( ! $attachment ) {
-			return false;
-		}
-
-		$attachment->guid           = $info['original'];
-		$attachment->post_mime_type = 'video/videopress';
-
-		wp_update_post( $attachment );
-
-		// Update the vp guid and set it to a direct meta property.
-		update_post_meta( $id, 'videopress_guid', $info['guid'] );
-
-		$meta = wp_get_attachment_metadata( $id );
-
-		$meta['width']             = $info['width'];
-		$meta['height']            = $info['height'];
-		$meta['original']['url']   = $info['original'];
-		$meta['videopress']        = $info;
-		$meta['videopress']['url'] = 'https://videopress.com/v/' . $info['guid'];
-
-		// Update file statuses
-		if ( ! empty( $format ) ) {
-			$meta['file_statuses'][ $format ] = $status;
-		}
-
-		if ( ! get_post_meta( $id, '_thumbnail_id', true ) ) {
-			// Update the poster in the VideoPress info.
-			$thumbnail_id = videopress_download_poster_image( $info['poster'], $id );
-
-			if ( is_int( $thumbnail_id ) ) {
-				update_post_meta( $id, '_thumbnail_id', $thumbnail_id );
-			}
-		}
-
-		wp_update_attachment_metadata( $id, $meta );
-
-		videopress_update_meta_data( $id );
-
-		// update the meta to tell us that we're processing or complete
-		update_post_meta( $id, 'videopress_status', videopress_is_finished_processing( $id ) ? 'complete' : 'processing' );
-
-		return true;
-	}
-
-	/**
-	 * Update poster image for a VideoPress media item.
-	 *
-	 * @param array $request The media item to update.
-	 * @return bool
-	 */
-	public function update_poster_image( $request ) {
-		$this->authenticate_user();
-
-		$post_id = $request['post_id'];
-		$poster  = $request['poster'];
-
-		$attachment = get_post( $post_id );
-		if ( ! $attachment ) {
-			return false;
-		}
-
-		$poster = apply_filters( 'jetpack_photon_url', $poster );
-
-		$meta                         = wp_get_attachment_metadata( $post_id );
-		$meta['videopress']['poster'] = $poster;
-		wp_update_attachment_metadata( $post_id, $meta );
-
-		// Update the poster in the VideoPress info.
-		$thumbnail_id = videopress_download_poster_image( $poster, $post_id );
-
-		if ( ! is_int( $thumbnail_id ) ) {
-			return false;
-		}
-
-		update_post_meta( $post_id, '_thumbnail_id', $thumbnail_id );
-
-		return true;
+		_deprecated_function( __METHOD__, 'jetpack-$$next-version$$', 'Automattic\Jetpack\VideoPress\XMLRPC' );
+		return self::init();
```

Comment:
```suggestion
		return XMLRPC::init();
```

I think this is a typo :) 
Also, are we planning to remove this whole file in a future version?

Reply:
thanks.

Yes, I'm not removing it now just to be extra safe, but not reason to keep it